arduino/smartcar_mqtt/smartcar_mqtt.ino

```diff
@@ -20,14 +23,17 @@ const auto echoPin = 7;
 const auto maxDistance = 400;
 SR04 front(arduinoRuntime, triggerPin, echoPin, maxDistance);
 
+std::vector<char> frameBuffer;
```

Comment:
Isn't there a convention to use `unsigned char` when you want to use bytes until `std::byte` is made available?

Reply:
In theory yes, in practice the MQTT message already stores `char`s